pkg/chunk/storage/factory.go

```diff
@@ -265,11 +265,17 @@ func NewBucketClient(storageConfig Config) (chunk.BucketClient, error) {
 // NewObjectClient makes a new StorageClient of the desired types.
 func NewObjectClient(name string, cfg Config) (chunk.ObjectClient, error) {
 	switch name {
+	case "aws", "s3":
```

Comment:
Given we're adding the support now, I would suggest to not support the ambiguous name `aws`, unless this results into any issue (if so, which one?):

```suggestion
	case "s3":
```

Reply:
The reason for keeping it like this is because we support both `aws` and `s3` for creating object client for chunks in S3 and I want to keep it consistent because for boltdb shipper we are going to pick up the type of storage that is configured for storing chunks. Does that make sense?

Reply:
Yes, consistency is a good argument in favor of it.